docs/netdata-cloud/node-membership-rules.md

```diff
@@ -1,4 +1,4 @@
-# Node Membership Rules
+# Node Membership Rules (Dynamic Room Allocation)
```

Comment:
We could use:

- "Dynamic Node Rules" <- a bit less to the point
- "Dynamic Node **grouping** Rules"

We definitely do not "allocate rooms" and nodes having "memberships", the word membership is not related to the word room, it is not like house -> room...

Again, if you just want to merge @sashwathn let me know to approve, but I would like us to solve the issue rather than giving the thing two names

Reply:
Also **Rule Based Room Assignment** could work